### ISSUE TITLE
client.js isConnected is a function in OpenAI version

### ISSUE THREAD

**Issue by SuperNiceDev** [transitive-bullshit/openai-realtime-api#1]
Hi, nice work!

But it seems isConnected is a function in OpenAI version.

https://github.com/openai/openai-realtime-api-beta/blob/cd8a9251dcfb0cba0d7b0501e9ff36c915f5090f/lib/client.js#L370
vs:
https://github.com/transitive-bullshit/openai-realtime-api/blob/f9cb1befd4ddd313342e9256a7a9edcfffaecbea/src/client.ts#L276

Best
Philipp

**Comment by transitive-bullshit** [comment]
Yep; I felt like this was one area where it didn't make sense for it to be a function, so 99% drop-in replacement instead of 100%.

Happy to reconsider if there's enough pushback && thanks! 🙏

**Comment by longseespace** [comment]
> Yep; I felt like this was one area where it didn't make sense for it to be a function, so 99% drop-in replacement instead of 100%.
> 
> Happy to reconsider if there's enough pushback && thanks! 🙏

Would be nice to add this notice to README. I just ran into this not sure what I was doing wrong 😅

**Comment by arjunlall** [comment]
Agreed, if the readme mentions drop in replacement it would be ideal if it worked without any changes.
